static/src/javascripts-legacy/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -131,17 +134,22 @@ define([
     }
 
     function ContributionsABTestVariant(options, test) {
+        var trackingCampaignId = test.epic ? 'epic_' + test.campaignId : test.campaignId;
+
         this.campaignId = test.campaignId;
         this.id = options.id;
         this.maxViews = options.maxViews || maxViews;
         this.isUnlimited = options.isUnlimited || false;
 
-        this.contributeURL = options.contributeURL || this.makeURL(contributionsURL, test.contributionsCampaignPrefix);
-        this.membershipURL = options.membershipURL || this.makeURL(membershipURL, test.membershipCampaignPrefix);
+        this.pageviewId = (config.ophan && config.ophan.pageViewId) || 'not_found';
```

Comment:
this was previously being computed inside the `makeURL` function, I moved it here as part of a refactor (as well as passing in the campaign code instead of computing it)